.npmignore

```diff
@@ -1 +1,17 @@
-demos/*
\ No newline at end of file
+bower_components
+.sass-cache
+build
+demos
+demos1
+TODO.md
+test
+testlib
+docs
+dist
```

Comment:
It is better to keep the dist folder.
Some people may prefer to add those files in script tags instead of using browserify/webpack